### PR TITLE
added download data wizard

### LINKED ISSUE #24: Create wizard for downloading the project
See the wireframe below for guidance:
project_download_wizard

### PR DESCRIPTION
**Pull request by meomancer**
this fix #24 
download layer data based on organization that selected

screenshots:
screenshot from 2016-12-21 16 06 43
screenshot from 2016-12-21 16 06 51
screenshot from 2016-12-21 16 07 01
screenshot from 2016-12-21 16 16 09

demo:
https://drive.google.com/file/d/0B7OxCB3EJZ5cYUtsZWxxb0dHWlk/view?usp=sharing

### CODE REVIEW

File: cadasta/gui/tools/cadasta_project_download_step_2.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+        :param geojson: geojson that will be saved
+        :type geojson: JSON object
+        """
+        filename = get_path_data(organization_slug, project_slug)
+        file_ = open(filename, 'w')
+        file_.write(json.dumps(geojson, sort_keys=True))
+        file_.close()
+        vlayer = QgsVectorLayer(
+            filename, "%s/%s" % (organization_slug, project_slug), "ogr")
```
Comment (ismailsunni) [inline cadasta/gui/tools/cadasta_project_download_step_2.py:165]: Use os.path.join since the path separator will be different in Windows and Unix

Comment (ismailsunni) [reply cadasta/gui/tools/cadasta_project_download_step_2.py:165]: Sorry, ignore this comment :D

**Comment by meomancer** [comment]
Trying to change using stacked widget

**Comment by timlinux** [comment]
Wow looking good @meomancer.

**Comment by meomancer** [comment]
I'm going to merge it
Will make PR with using stacked wizard again dimas's PR

**Pull request merged**